api/service/blockproposal/service.go

```diff
@@ -22,38 +22,26 @@ func New(readySignal chan consensus.ProposalType, commitSigsChan chan []byte, wa
 	return &Service{readySignal: readySignal, commitSigsChan: commitSigsChan, waitForConsensusReady: waitForConsensusReady}
 }
 
-// StartService starts block proposal service.
-func (s *Service) StartService() {
+// Start starts block proposal service.
+func (s *Service) Start() error {
 	s.stopChan = make(chan struct{})
 	s.stoppedChan = make(chan struct{})
 
-	s.Init()
-	s.Run(s.stopChan, s.stoppedChan)
-}
-
-// Init initializes block proposal service.
-func (s *Service) Init() {
+	s.run(s.stopChan, s.stoppedChan)
+	return nil
```

Comment:
if there is no error to return, just remove the `error` return type.

Reply:
This `Service` type implements `Service` interface, which requires the signature to return an error. This error is returned for sync protocol which will be added in some future PR.